testpackage/tests/Flowthrough_damr/Flowthrough_damr.cfg

```diff
@@ -0,0 +1,113 @@
+ParticlePopulations = proton
+
+project = Flowthrough
+propagate_field = 1
+propagate_vlasov_acceleration = 1
+propagate_vlasov_translation = 1
+dynamic_timestep = 1
+
+[proton_properties]
+mass = 1
+mass_units = PROTON
+charge = 1
+
+[AMR]
+max_spatial_level = 2
+adapt_refinement = 1
+use_alpha1 = 1
+use_alpha2 = 1
+refine_cadence = 2
+
+[gridbuilder]
+x_length = 16
+y_length = 8
+z_length = 8
+x_min = -8e7
+x_max = 8e7
+y_min = -4e7
+y_max = 4e7
+z_min = -4e7
+z_max = 4e7
+t_max = 182.0
+dt = 2.0
+
+[proton_vspace]
+vx_min = -1.92e6
+vx_max = +1.92e6
+vy_min = -1.92e6
+vy_max = +1.92e6
+vz_min = -1.92e6
+vz_max = +1.92e6
+vx_length = 16
+vy_length = 16
+vz_length = 16
```

Comment:
So this is a low block content. How long does this take to run on Carrington?

Reply:
Well on Mahti it's 66s, so about 150% of Flowthrough_amr

Reply:
Hmm. And if gridbuilder.t_max is increased to 722, it'll be at least 4x or more like 5x that. :-/

Reply:
Yep, around that. Letting the pulse propagate the entire box might not be that important for this test, my aim was just to have a test that both refines and coarsens cells dynamically, which it does in the shorter time.